### ISSUE TITLE
Error in GPX.maxSpeed()

### ISSUE THREAD

**Issue by trigchen** [melato/gpx#1]
GPX.maxSpeed() is same as GPX.duration().

**Comment by melato** [comment]
Yes, you are right.  It turns out I never used it.  I'd rather remove it because it's not clear what it should do.  Speed can be calculated in two different ways:  a)  The GPX track waypoints may have a speed property, as a GPX extension.  b)  By dividing the difference in distance over the difference in time from the previous point.  What do you expect maxSpeed() to do?

**Comment by trigchen** [comment]
The b) should be OK.
I've cloned your git repositories(gpx, gps, xml) and made some
modifications.
Please see https://github.com/trigchen/gpx.

Best regards,

Trig

2016-06-26 23:24 GMT+08:00 melato notifications@github.com:
